src/main/java/net/fabricmc/stitch/commands/CommandProposeFieldNames.java

```diff
@@ -40,42 +33,34 @@ public CommandProposeFieldNames() {
 
     @Override
     public String getHelpString() {
-		return "<input jar> <input mappings> <output mappings> [<input jar namespace>] [<output namespace>]";
+        return "<input jar> <input mappings> <output mappings>";
```

Comment:
Why remove the name spaces?

Reply:
I just reverted it to 0.2.60, which is what loom has been using for 0.2.5. No point trying to fix whatever changed in 0.2.61 since we don't need v1 anymore...

Reply:
Ah ok, that makes sense then. Thats fine.